src/test/java/org/minions/devfund/lourdes/MovieDatabaseTest.java

```diff
@@ -0,0 +1,151 @@
+package org.minions.devfund.lourdes;
+
+import org.junit.Before;
+import org.junit.Test;
+
+import java.util.List;
+
+import static org.junit.Assert.assertEquals;
+
+/**
+ * Implemented tests for {@link MovieDatabase}.
+ */
+public class MovieDatabaseTest {
+    private static final String MOVIE_NAME_1 = "Sexto sentido";
+    private static final String MOVIE_NAME_2 = "Duro de Matar";
+    private static final String[] ACTORS_1 = new String[]{"Bruce Willis", "Haley Joel"};
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Define a constant instead of duplicating this literal "Bruce Willis" 3 times. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1192)